docs/training-apis/api-core/distributed.txt

```diff
@@ -21,25 +21,41 @@ considerations are:
 #. Create a ``4_distributed.py`` training script by copying the ``3_hpsearch.py`` from
    :ref:`core-hpsearch`.
 
-#. In your ``if __name__ == "__main__"`` block, add ``multiprocessing`` code to launch one process
-   per slot.
+#. Add launcher logic to execute one worker subprocess per slot.
+
+   Typically, your distributed training framework provides a launcher, as Horovod provides
+   ``horovodrun`` and Torch provides ``torch.distributed.launch``, so you do not have to write this
+   launcher yourself.
+
+   Start with a ``launcher_main()`` function that executes one worker subprocess per slot.
 
    .. literalinclude:: ../../../examples/tutorials/core_api/4_distributed.py
       :language: python
       :dedent:
-      :start-at: slots_per_node = len(info.slot_ids)
+      :pyobject: launcher_main
 
-   Typically, your distributed training framework provides a launcher, as Horovod provides
-   ``horovodrun`` and Torch provides ``torch.distributed.launch``, so you do not have to write the
-   launcher yourself.
+   Even when using a third-party provided launcher, you will still need to wrap your worker script
+   in the ``python -m determined.launcher.wrap_rank`` wrapper script for the WebUI log viewer to
+   filter logs by rank. This wrapper is automatically included when using the Determined-provided
+   launchers.
 
-   Even if the launcher is provided, you always need to create a
-   :class:`~determined.core.DistributedContext` on each worker to pass to ``core.init()``. In the
-   absence of a distributed training framework that might define rank/local_rank/cross_rank, you can
-   derive those values from the ClusterInfo API, as in this example.
+   Also add a ``worker_main()`` that will run training on each slot:
```

Comment:
Change to: " Also, add a ``worker_main()`` function that runs training on each slot:"
[maybe 'method' is more accurate than 'function']

Reply:
There's no universal definition, but this [widely-upvoted stackoverflow post](https://stackoverflow.com/a/155633) reflects the definitions I am also most familiar with.